apps/ledger-live-mobile/src/components/BleDevicePairingFlow/index.tsx

```diff
@@ -205,21 +208,40 @@ const BleDevicePairingFlow: React.FC<BleDevicePairingFlowProps> = ({
     requestToSetHeaderOptions,
   ]);
 
+  const onLocked = useCallback(
+    (device: Device) => {
+      onRetryPairingFlow();
```

Comment:
You do this to go back to the scanning step right ?

But so currently there is no automatic way to see that the device is not locked anymore (no retrying on the pairing step directly). The user has to click on cross button to exit. Maybe there is a way to update the `useBleDevicePairing` so it retries on this specific locked device error - and you don't need to restart the entire flow

Reply:
Done